packages/replay-next/components/errors/UnexpectedErrorForm.tsx

```diff
@@ -31,7 +31,11 @@ export function UnexpectedErrorForm({
       currentUserName={currentUserName}
       details={
         <>
-          {details && <div className={styles.Details}>{details}</div>}
+          {details && (
+            <div className={styles.Details} data-test-id="UnexpectedErrorDetails">
+              {details}
+            </div>
+          )}
```

Comment:
Putting the test id here seems a bit more fragile than putting it on the root of the unexpected error dialog, because it's possible that our helper would miss an unexpected fatal error if it didn't have a details string.

Reply:
I'll check for the visibility of the support form first and then extract the details if they are available (and otherwise throw a "Recording crashed" error without details).

Reply:
Nice.